### PR TITLE
[ANCHOR-749] Add SEP-38 validation documentation

### PR DESCRIPTION
**Pull request by lijamie98**
*** Recreated due to operation mistake

Many validation rules are added to the response of the /rate endpoint. This PR adds the validations that may not be straight forward to the endpoint developers.

### CODE REVIEW

File: openapi/anchor-platform/main-callbacks.yaml [hunk 1]
```diff
@@ -208,6 +208,26 @@ paths:
...
         either `sell_delivery_method` and `buy_delivery_method` may be included in requests, but never both, since either
         `sell_asset` or `buy_asset` is a Stellar asset.
+        
+        
+        Upon receiving the response, the Anchor Platform will validate the amount and price of the response.
+
+        If the validation fails, the Platform will respond to the client application's request with a HTTP status
+        code of `502 Bad Gateway`.
+
+        The `sell_mount`, `buy_amount`, `price`, and `fee` are validated as follows:
```
Comment (philipliu) [inline openapi/anchor-platform/main-callbacks.yaml:218]: typo: `sell_amount`

File: openapi/anchor-platform/main-callbacks.yaml [hunk 2]
```diff
@@ -208,6 +208,26 @@ paths:
...
+
+        - if `rate.fee` exists,
+          - `rate.fee.asset` must be defined with proper `significant_decimals` in the asset configuration.
```
Comment (philipliu) [inline openapi/anchor-platform/main-callbacks.yaml:221]: What does proper mean here?

**Review by philipliu — COMMENTED** [review]
I think we need to run `yarn api` to generate the MDX files too.

File: openapi/anchor-platform/main-callbacks.yaml (hunk 2 above)
Comment (lijamie98) [reply openapi/anchor-platform/main-callbacks.yaml:221]: Exists and positive.

File: platforms/stellar-disbursement-platform/api-reference/list-all-disbursement-receivers.api.mdx [hunk 3]
```diff
@@ -718,7 +718,7 @@ Fetches a list of receivers within a specific disbursement using the disbursemen
                                     required={false}
```
Comment (philipliu) [inline platforms/stellar-disbursement-platform/api-reference/list-all-disbursement-receivers.api.mdx:718]: nit: We should remove SDP changes from the commit next time.

**Review by philipliu — APPROVED** [review]

**Pull request merged**